src/infra/dataverse/sparql/sparqlGateway.ts

```diff
@@ -58,16 +58,23 @@ export const sparqlGateway: DataversePort = {
         ?id rdf:type ?type .
         ?type rdf:type owl:Class .
         ?metadata core:describes ?id .
-        ?metadata core:hasTitle ?title .
-        FILTER langMatches(lang(?title),'${language}')
+        OPTIONAL { 
+          ?metadata core:hasTitle ?title .
+          FILTER langMatches(lang(?title),'${language}')
+        }
+        OPTIONAL { 
+          ?metadata core:hasTitle ?title .
+        }
```

Comment:
Seems redundant. Is it required?

Reply:
If the language filter returns nothing, this ensures a fallback value, if there is no fallback value and the value is not found, then it returns no result.

Reply:
:thinking: Did you try it? IMHO, this current query might not perform as expected. It first focuses on a particular language, but then it considers all languages on the second otional. This means it could return titles in any language, not just the one we're interested in.

I guess you could have three separate optional sections. One for the language you're interested in, one for english (`en`), and one for no language. And you can smartly design your SPARQL template to skip the english section when you're already targeting english.